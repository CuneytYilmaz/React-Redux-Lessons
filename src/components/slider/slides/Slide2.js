import React from 'react';

import Button from '../../ui/Button';

import image from '../../../img/slides/slider-image-2.svg';

const Slide2 = () =>
  <div className="row Slider__slide">
    <div className="col-md-6">
      <h1 className="Slider__title Slider__title--main">
        Kategorini seç, senin için seçtiğimiz videonun keyfini çıkar!
      </h1>
      <p className="u-gap-bottom">
        Video izlemeye başlamak için ilk önce kategorini seç ve rastgele seçtiğimiz videonun keyfini
        çıkar. Eğer videoyu beğenmezsen sıradaki video’ya geç.
      </p>
      <Button>Kullanmaya Başla!</Button>
    </div>
    <div className="col-md-6 u-text-center">
      <img
        src={image}
        className="u-inline-block"
        width="400"
        height="378"
        alt="Kategorini seç, senin için seçtiğimiz videonun keyfini çıkar!"
      />
    </div>
  </div>;

export default Slide2;
