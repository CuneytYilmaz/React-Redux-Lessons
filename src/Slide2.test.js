import React from 'react';
import { shallow } from 'enzyme';

import Slide2 from './components/slider/slides/Slide2';

it('renders correctly', () => {
  const wrapper = shallow(<Slide2 />);

  expect(wrapper).toMatchSnapshot();
});
