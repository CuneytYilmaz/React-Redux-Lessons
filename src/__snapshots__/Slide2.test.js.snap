// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row Slider__slide"
>
  <div
    className="col-md-6"
  >
    <h1
      className="Slider__title Slider__title--main"
    >
      Kategorini seç, senin için seçtiğimiz videonun keyfini çıkar!
    </h1>
    <p
      className="u-gap-bottom"
    >
      Video izlemeye başlamak için ilk önce kategorini seç ve rastgele seçtiğimiz videonun keyfini çıkar. Eğer videoyu beğenmezsen sıradaki video’ya geç.
    </p>
    <Button
      primary={true}
      secondary={false}
    >
      Kullanmaya Başla!
    </Button>
  </div>
  <div
    className="col-md-6 u-text-center"
  >
    <img
      alt="Kategorini seç, senin için seçtiğimiz videonun keyfini çıkar!"
      className="u-inline-block"
      height="378"
      src="slider-image-2.svg"
      width="400"
    />
  </div>
</div>
`;
